src/Utils/StyleUtils.ts

```diff
@@ -0,0 +1,23 @@
+// Adapted from https://gist.github.com/davidgilbertson/ed3c8bb8569bc64b094b87aa88bed5fa
+export function copyStyles(sourceDoc: Document, targetDoc: Document): void {
```

Comment:
Interesting -- what's the motivation for injecting the parent page's styles to the iframe?

Reply:
I noticed that there were some output transforms (like Nteract Data Explorer, StreamText etc) which depend on styles which were present in the document. May be the right thing to do here is to make sure the output transforms inject their own styles when rendered?

Reply:
I did some more investigation and looks like Nteract Data Explorer is indeed injecting its own CSS styles. And still we don't see those styles in the `<iframe>` by default. This leads me to believe that React portals are injecting the styles in parent document.